lib/config.go

```diff
@@ -551,24 +570,42 @@ func LogOccConfigs() {
 			"max_desire_healthy_worker_pct":        gAppConfig.MaxDesiredHealthyWorkerPct,
 		},
 		"R-W-SPLIT": {
-			"readonly_children_pct":          gAppConfig.ReadonlyPct,
-			"cfg_from_tns_override_rw_split": gAppConfig.CfgFromTnsOverrideRWSplit,
+			"readonly_children_pct": gAppConfig.ReadonlyPct,
 		},
 		"RAC": {
 			"management_table_prefix": gAppConfig.ManagementTablePrefix,
 			"rac_sql_interval":        gAppConfig.RacMaintReloadInterval,
 			"rac_restart_window":      gAppConfig.RacRestartWindow,
 		},
-		"NO-CATEGORY": {
+		"GENERAL-CONFIGURATIONS": {
 			"database_type":   gAppConfig.DatabaseType, //	Oracle = 0; MySQL=1; POSTGRES=2
-			"cfg_from_tns":    gAppConfig.CfgFromTns,
 			"log_level":       gOpsConfig.logLevel,
 			"high_load_pct":   gAppConfig.HighLoadPct,
 			"init_limit_pct":  gAppConfig.InitLimitPct,
 			"num_standby_dbs": gAppConfig.NumStdbyDbs,
 		},
+		"TNS": {
```

Comment:
can we change the future name as "ENABLE_CFG_FROM_TNS" or better alternative name.

Reply:
Addressed